admin/extract_docstrings.py

```diff
@@ -0,0 +1,45 @@
+# GemRB - Infinity Engine Emulator
+# Copyright (C) 2020 The GemRB Project
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+import GemRB
+
+import inspect
+import tempfile
+import os
+from subprocess import Popen, PIPE
+
+
+def adjust_document(document, **kwargs):
+	# Pass through Pandoc.
+	# Add an appropriately filled module field.
+	# Add the link to the function index.
+	metadata = ["--metadata={}:{}".format(kwname, kwvalue)
+		for kwname, kwvalue in kwargs.items()]
+	pandoc_invocation = ["pandoc", "--from=dokuwiki",
+			"--to=markdown+yaml_metadata_block", "--standalone"] + metadata
+	pipe = Popen(pandoc_invocation, stdin=PIPE, stdout=PIPE)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=5396854)